h/buildext.py

```diff
@@ -295,11 +303,6 @@ def check_sentry_dsn_is_public(dsn):
                     '"wss://hypothes.is/ws")',
                     dest='websocket_url',
                     metavar='URL')
-parser.add_argument('--no-bundle-sidebar',
```

Comment:
I've taken out the option to skip bundling the sidebar for the moment to simplify things. After this is merged, it should be a fairly straightforward addition to move to a workflow where gulp can incrementally build the assets directly into the `build/chrome/{lib, public}` folders, post a notification to Chrome when an update happens and the extension will auto-reload itself.